src/database/services/trello-api-service.ts

```diff
@@ -0,0 +1,202 @@
+import fetch from "node-fetch";
+
+/**
+ * Class that implements Trello Service for card operations.
+ */
+export class TrelloService {
+  private readonly apiKey = process.env.TRELLO_API_KEY;
+  private readonly apiToken = process.env.TRELLO_TOKEN;
+  private readonly baseUrl = 'https://api.trello.com/1';
+  private readonly boardId = process.env.TRELLO_MANAGEMENT_BOARD_ID;
+
+  /**
+   * Gets all boards from Trello
+   *
+   * @returns Trello board
+   */
+  public async getBoardList(): Promise<any[]> {
```

Comment:
Do we not have any typings for the Trello API?

Reply:
No, there is no official libraries with types for Trello API

Reply:
Surprising but it seems to be the case. I think there may be some places it is worth using custom types but will try to point some of these out in comments.

Reply:
but there is an OpenAPI spec: https://dac-static.atlassian.com/cloud/trello/swagger.v3.json?_v=1.573.0